src/charmed_kubeflow_chisme/exceptions/_error_status_with_message.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+class UnitErrorStatusWithMessage(Exception):  # noqa: N818
```

Comment:
what is about `JujuUnitError` ?

Reply:
@beliaev-maksim I think adding the `Juju` prefix in a charm library may not be the best idea, it could be mistaken with a real juju error.